src/generator_c.c

```diff
@@ -35,3 +36,56 @@ NM_GENERATOR_(_test) {
 
     #undef NM_ERR_RET
 }
+
+NM_GENERATOR_(kfmon) {
+    #define NM_ERR_RET NULL
+
+    // Default with no arg or an empty arg is to request a gui-listing
+    char *kfmon_cmd = NULL;
+    if (!arg || !*arg || strcmp(arg, "gui") == 0) {
+        kfmon_cmd = strdupa("gui-list");
+    } else if (strcmp(arg, "all") == 0) {
+        kfmon_cmd = strdupa("list");
```

Comment:
Are you sure the `strdupa` is required here?

Reply:
That's a good point, I could assign a string literal here and it'd essentially point to that string in .rodata, right?

Reply:
Yes, and you'd need to make the var a `const char*` too.

Reply:
Also, to match with the rest of the code, can you use `!strcmp` instead of `== 0`?

Reply:
Ah, GH comments mysteriously not refreshing automatically, I'll take care of the strcmp syntax ;).